server/controllers/UsersController.js

```diff
@@ -1,8 +1,18 @@
 import bcrypt from 'bcrypt-nodejs';
+import { Op } from 'sequelize';
+import cloudinary from 'cloudinary';
+import winston from 'winston';
+
 import models from '../models';
-import createToken from '../helpers/createToken';
 import sendEmail from '../helpers/sendEmail';
 import verifyEmailMessage from '../helpers/verifyEmailMessage';
+import { createToken } from '../helpers/tokenUtils';
+
+cloudinary.config({
```

Comment:
Why is this Cloudinary config object placed in the UserController? I believe we have a config folder for this purpose.

Reply:
Oh yeah, this is true. I will work on that. Thanks for pointing that out.